server/etcdserver/server.go

```diff
@@ -382,6 +383,7 @@ func NewServer(cfg config.ServerConfig) (srv *EtcdServer, err error) {
 	be := openBackend(cfg, beHooks)
 	ci.SetBackend(be)
 	cindex.CreateMetaBucket(be.BatchTx())
+	dbversion.UpdateLocalVersion(be.BatchTx(), version.Version)
```

Comment:
I think its not that easy ... the version of the file (if it needs to be meanigful) can be updated if we make sure the file conforms to v3.6 'storage changes'. 

E.g. 
  meta/term
  meta/snapshot
  meta/version 
In v3.6 format must be set. Only when we confirm/guarantee this 'invariants' file can be called v3.6.

Reply:
I don't think there is place where we confirm all those invariants as it would require us to start maintaining schema (would be good to start thinking about this). Moved to precommit hooks, to execute once before first commit. At this point all fields should be set based on assumption that "storage saved by version X is in version X"

Reply:
Thank you. Commented in the new location.